### DISCUSSION TITLE
Cybernetic limitation

### DISCUSSION THREAD

**Discussion by TheDoktorZ** [unitystation/unitystation#7084] [Ideas]
### Description

This is **not** a new **idea**, but a request to make something against the abused cybernetic powers. Currently there is no limitation how many cybernetic body parts can be attached to someone's body. Some madmen abuses this and attaches like a dozen leg and a dozen arm to themselves. Every leg gives extra speed while walking and running, every arm gives extra speed while crawling. Some people abuse this so hardly they can reach super-sonic speed while walking.

I think the legs should be capped by **4** and the arms by **6** in total, including the original, organic arms. Of course those could be still removed and replaced with cybernetic counterparts.

**Comment by PerfectTangent** [comment]
The plan as discussed in #health is to cap the number of limbs that can be attached to a body, probably to 4 or 8 limbs total, of any combination.

**Comment by PerfectTangent** [comment]
I'm speaking to just default humans. Other species may get a different allotment of maximum limbs.

**Comment by corp-0** [comment]
I'm making this issue a discussion as it feels more appropriate.
